shared/src/Datadog.Trace.ClrProfiler.Native/constants.h

```diff
@@ -0,0 +1,51 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2022 Datadog, Inc.
+
+#pragma once
+
+#include "../../../shared/src/native-src/string.h"
+
+inline static const std::string conf_filename = "loader.conf";
+inline static const ::shared::WSTRING datadog_logs_folder_path = WStr(R"(Datadog .NET Tracer\logs)");
+
+// Note that you should also consider adding to the SSI tracer/build/artifacts/requirements.json file
+// FIXME: this should also take into account case insensitivity, but that is not yet supported
+// https://devblogs.microsoft.com/oldnewthing/20241007-00/?p=110345
+const shared::WSTRING default_exclude_processes[]{
+    WStr("aspnet_compiler.exe"),
+    WStr("aspnet_state.exe"),
+    WStr("CollectGuestLogs.exe"), // https://github.com/Azure/WindowsVMAgent
+    WStr("csc.exe"),
+    WStr("createdump"),
```

Comment:
We're trying to keep them in alphabetical order
```suggestion
    WStr("createdump"),
    WStr("csc.exe"),
```